forge-gui/res/cardsfolder/upcoming/gollum_scheming_guide.txt

```diff
@@ -0,0 +1,20 @@
+Name:Gollum, Scheming Guide
+ManaCost:1 B
+Types:Legendary Creature Halfling Horror
+PT:2/1
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigPeek | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME attacks, look at the top two cards of your library, put them back in any order, then choose land or nonland. An opponent guesses whether the top card of your library is the chosen kind. Reveal that card. If they guessed right, remove NICKNAME from combat. Otherwise, you draw a card and NICKNAME can't be blocked this turn.
+SVar:TrigPeek:DB$ RearrangeTopOfLibrary | Defined$ You | NumCards$ 2 | SubAbility$ DBChoosePlayer
+SVar:DBChoosePlayer:DB$ ChoosePlayer | Defined$ You | Choices$ Player.Opponent | ChoiceTitle$ Choose an opponent | SubAbility$ DBChoose
+SVar:DBChoose:DB$ GenericChoice | Choices$ ChooseLand,ChooseNonland | Defined$ You
+SVar:ChooseLand:DB$ GenericChoice | Defined$ ChosenPlayer | Choices$ GuessLand,GuessNonland | SpellDescription$ Choose land.
+SVar:ChooseNonland:DB$ GenericChoice | Defined$ ChosenPlayer | Choices$ GuessLand,GuessNonland | SpellDescription$ Choose nonland.
+SVar:GuessLand:DB$ PeekandReveal | PeekAmount$ 1 | RevealValid$ Card.Land | RememberRevealed$ True | SubAbility$ DBBranch | SpellDescription$ Guess land.
```

Comment:
your logic is wrong:
the opponent doesn't just guess the type, they must guess if you lied or not
so you probably need even more subabilities :P

Reply:
So it doesn't matter what the top card actually is ?

Reply:
no, that matters too

Reply:
After thinking about it a bit more you could probably go back to the previous version
Just make sure to include `ShowChoice$ ExceptSelf` on the first Generic (for flavor)